README.md

```diff
@@ -1,24 +1,139 @@
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* Ruby version
+- [📝 License](#license)
 
-* System dependencies
+# 💰 Greetings App Back-end <a name="about-project"></a>
 
-* Configuration
+> This is a starting point to create a React front-end with a Rails back-end and connect them to display a random message
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://rspec.info/">Rspec</a>
 
-* How to run the test suite
+## Link to the Front-end repository
 
-* Services (job queues, cache servers, search engines, etc.)
+[Hello_rails_front-end](https://github.com/Chaseknis/helllo_rails_front-end.git)
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- You can be able to access data in the api which are randomly displayed using the front-end react app and connecting it to this app serving as a back end to store all the data in the Api
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Command line or terminal
+- IDE such as VsCode.
+- Rails needs to be installed on your system.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example command:
+
+```sh
+  git clone https://github.com/Chaseknis/hello_rails_back-end.git
+  cd hello_rails_back-end
+
+```
+
+### Install
+
+Example command:
+
+```sh
+  cd hello_rails_back-end
+  bundle install (install all dependencies)
+
+```
+
+### Usage
+
+To run the project redirecting to the root directory of the project and run the following command:
+
+```sh
+ bin/rails server (to run web application)
+
+```
+
+### API Documentation
+
+Run the following command:
+
+```sh
+ rails s
+
+```
+and go to this link in your browser
+<http://localhost:3000/api/random_greeting>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Innocent N.**
+
+- GitHub: [@Chaseknis](https://github.com/Chaseknis)
+- Twitter: [@Chaseknis](https://twitter.com/Chasetotheworld)
+- LinkedIn: [Innocent N.](https://www.linkedin.com/in/innocent-n/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [UX Desktop design]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Chaseknis/hello_rails_back-end/issues).
+
```

Comment:
- [OPTIONAL] When I click on the [issues page] link, I'm getting the 404 Error. Kindly fix this by adding the correct issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

Reply:
Fixed